frontend/src/app/App.js

```diff
@@ -208,165 +205,204 @@ export function App({ initialLoading, isLoginRequired }) {
-                )}
-              </PrivatePage>
-
-              <Page path="/admin/:activeMenu?" title={t`Admin`}>
-                {isLoggedIn() && isEmailValidated() && currentTFAMethod() !== 'NONE' ? (
-                  <AdminPage />
-                ) : (
-                  <Redirect
-                    to={{
-                      pathname: '/sign-in',
-                      state: { from: location },
-                    }}
-                  />
-                )}
-              </Page>
-
-              <PrivatePage isLoginRequired={isLoginRequired} path="/domains" title={t`Domains`} exact>
-                {() => (
-                  <ErrorBoundary FallbackComponent={ErrorFallbackMessage}>
+                element={
+                  <PrivatePage condition={(isLoggedIn() && isEmailValidated()) || !isLoginRequired} setTitle={false}>
+                    <OrganizationDetails />
+                  </PrivatePage>
+                }
+              />
+
+              <Route
+                path="/admin/:activeMenu?"
+                element={
+                  <PrivatePage
+                    condition={isLoggedIn() && isEmailValidated() && currentTFAMethod() !== 'NONE'}
+                    title={t`Admin`}
+                  >
+                    <AdminPage isLoginRequired={isLoginRequired} />
+                  </PrivatePage>
+                }
+              />
+
+              <Route
+                path="/domains"
+                exact
+                element={
+                  <PrivatePage condition={(isLoggedIn() && isEmailValidated()) || !isLoginRequired} title={t`Domains`}>
                     <DomainsPage />
-                  </ErrorBoundary>
-                )}
-              </PrivatePage>
+                  </PrivatePage>
+                }
+              />
 
-              <PrivatePage isLoginRequired={true} path="/domains/:domainSlug/:activeTab?" setTitle={false} exact>
-                {() => (
-                  <ErrorBoundary FallbackComponent={ErrorFallbackMessage}>
+              <Route
+                path="/domains/:domainSlug/:activeTab?"
+                exact
+                element={
+                  <PrivatePage condition={isLoggedIn() && isEmailValidated()} setTitle={false}>
                     <GuidancePage />
-                  </ErrorBoundary>
-                )}
-              </PrivatePage>
+                  </PrivatePage>
+                }
+              />
 
-              <PrivatePage
-                isLoginRequired={true}
+              <Route
                 path="/domains/:domainSlug/dmarc-report/:period?/:year?"
-                setTitle={false}
                 exact
-              >
-                {() => (
-                  <ErrorBoundary FallbackComponent={ErrorFallbackMessage}>
+                element={
+                  <PrivatePage condition={isLoggedIn() && isEmailValidated()} setTitle={false}>
                     <DmarcReportPage />
```

Comment:
The links for the DMARC report page on the DmarcByDomainPage are to `/dmarc-summaries/domains/domain/dmarc-report/month/year`. Links seem to be relative now if they don't have a leading `/`